src/components/editor/MonacoEditor.tsx

```diff
@@ -39,39 +39,36 @@ function MonacoEditor({
         null
     );
     const [isChanging, setIsChanging] = useState<boolean | null>(null);
-    const [hasServerChanges, setHasServerChanges] = useState<any | null>(null);
+    const [hasServerChanges] = useState<boolean>(false);
     const [showServerDiff, setShowServerDiff] = useState(false);
 
     const serverUpdate = useZustandStore<
         EditorStoreState<DraftSpecQuery>,
         EditorStoreState<DraftSpecQuery>['serverUpdate']
     >((state) => state.serverUpdate);
 
-    useEffect(() => {
-        if (editorRef.current) {
-            const currentStringValue = editorRef.current.getValue();
+    // TODO (sync editing)
+    // useEffect(() => {
+    //     if (editorRef.current) {
+    //         const currentStringValue = editorRef.current.getValue();
 
-            if (currentStringValue) {
-                const currentEditorValue = JSON.parse(currentStringValue);
+    //         if (currentStringValue) {
+    //             const currentEditorValue = JSON.parse(currentStringValue);
 
-                if (!isEqual(currentEditorValue, serverUpdate)) {
-                    setHasServerChanges(serverUpdate);
-                } else {
-                    setHasServerChanges(null);
-                }
-            }
-        }
+    //             if (!isEqual(currentEditorValue, serverUpdate)) {
+    //                 setHasServerChanges(serverUpdate);
+    //             } else {
+    //                 setHasServerChanges(null);
+    //             }
+    //         }
+    //     }
 
-        // eslint-disable-next-line react-hooks/exhaustive-deps
-    }, [serverUpdate]);
+    //     // eslint-disable-next-line react-hooks/exhaustive-deps
+    // }, [serverUpdate]);
 
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    const bar = useCallback(
+    const editorUpdateDebounce = useCallback(
```

Comment:
Since this function is not being exported elsewhere, what do you think about dropping the word _editor_ in the name? Not a critical change, but `updateDebounce` aligns with the function convention where names start with an active verb and it is slightly more concise.

Reply:
I'm going to leave it. The name has a label because later on I feel we're going to need other debouces/throttles in there that are not for the editor itself.